docs/source/tools/mythxvsc/index.rst

```diff
@@ -0,0 +1,61 @@
+.. meta::
+   :description: MythXvsc is an extension for running MythX smart contract analysis from VS Code. 
+   
+.. _tools.mythxvsc:
+
+VS Code
+=======
+
+MythXvsc is an extension for running MythX smart contract analysis from your VS Code.
+
+The extension provides:
+
+- Smart contract compilation (via solidity VSCode extension)
```

Comment:
`Smart contract compilation (via the Solidity VS Code extension)`

(Adding article and capitalization.)